src/sandstorm/app-index/app-index.c++

```diff
@@ -67,12 +67,13 @@ template <typename Context>
 void handleError(Context& context, kj::Exception&& e) {
```

Comment:
It's one of the greatest new features in C++11! Abstractly speaking, it means that ownership is being transferred without a copy.